xc/xenops_server_xen.ml

```diff
@@ -772,6 +774,9 @@ module HOST = struct
            hypervisor = {
              Host.version = xen_version_string;
              capabilities = xen_capabilities;
+           };
+           chipset_info = {
+             iommu = iommu;
```

Comment:
No need to change it, but just to let you know: you could drop the `= iommu` here.